ITechArt_project/urls.py

```diff
@@ -14,8 +14,14 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.contrib.staticfiles.urls import staticfiles_urlpatterns
+from django.urls import path, include
+from ITechArt_project import settings
```

Comment:
It's not a bug, but quite different from common codestyle in Django. All applications, folders etc should be in lower case and snake_case like `itechart_project` here